python/ray/serve/_private/deployment_state.py

```diff
@@ -2095,17 +2101,17 @@ def _check_startup_replicas(
                 self._deployment_scheduler.on_replica_running(
                     replica.replica_id, replica.actor_node_id
                 )
-                full_replica_startup_duration = time.time() - replica._start_time
-                scheduling_duration_s = (
-                    full_replica_startup_duration - replica.initialization_duration_s
+                e2e_replica_start_latency = time.time() - replica._start_time
+                scheduling_latency_s = (
+                    e2e_replica_start_latency - replica.initialization_latency_s
                 )
                 logger.info(
                     f"{replica.replica_id} started successfully "
                     f"on node '{replica.actor_node_id}' after "
-                    f"{full_replica_startup_duration:.1f}s. "
-                    f"Scheduling duration: {scheduling_duration_s:.1f}s.",
-                    "Initialization duration: "
-                    f"{replica.initialization_duration_s:.1f}s.",
+                    f"{e2e_replica_start_latency:.1f}s. "
+                    f"Scheduling latency: {scheduling_latency_s:.1f}s.",
+                    "Initialization latency: "
+                    f"{replica.initialization_latency_s:.1f}s.",
```

Comment:
I think we can make this more information-dense:
```suggestion
                    f"{e2e_replica_start_latency:.1f}s (replica constructor took {replica.initialization_latency_s:.1f}s})). "
```

Reply:
this does not lose any information (total latency minus constructor is scheduling latency)

Reply:
Sounds good, I modified the log a bit. Please take another look.